testgrid/tgapi/pkg/handlers/instance_sonobouy.go

```diff
@@ -41,7 +41,7 @@ func InstanceSonobuoyResults(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	if err := testinstance.SetInstanceFinishedAndSuccess(instanceID, isSuccess); err != nil {
+	if err := testinstance.SetInstanceFinishedAndSuccess(instanceID, isSuccess, "sonobuoy_results"); err != nil {
```

Comment:
why hardcode `sonobuoy_results` here?

Reply:
cause that is the cause of the failure

Reply:
right but it won't always be a failure, will it?